cmd/snap/main_test.go

```diff
@@ -80,6 +82,8 @@ func (s *BaseSnapSuite) RedirectClientToTestServer(handler func(http.ResponseWri
 
 type SnapSuite struct {
 	BaseSnapSuite
+
+	AuthFile string
```

Comment:
That's a very good point. Further: I think we should set the environ to point to the fake auth file in ` (*BaseSnapSuite).SetUpSuite` (and unset it in tear down), independently of whether the test needs to log in or not.